Build.fsx

```diff
@@ -69,7 +69,7 @@ Target "PatchAssemblyVersions" (fun _ ->
         buildVersion.fileVersion
         buildVersion.infoVersion
 
-    let filesToPatch = !! "Src/*/Properties/AssemblyInfo.*"
+    let filesToPatch = !! "Src/*/Properties/AssemblyInfo.fs"
```

Comment:
Yes, adding that comment can be helpful in the future because it [explains the *why*](https://ayende.com/blog/163297/the-importance-of-comments). (On the contrary, [explaining the *how*, can be an apology](http://butunclebob.com/ArticleS.TimOttinger.ApologizeIncode), of course.)